### DISCUSSION TITLE
无法复现svtr的论文精度，相同的配置文件，训练精度相差30%

### DISCUSSION THREAD

**Discussion by wuxiaolianggit** [PaddlePaddle/PaddleOCR#12400] [Q&A]
大佬们好，我这边按照svtr_tiny的配置文件，使用使用MJSynth和SynthText两个文字识别数据集训练，在IIIT, SVT, IC03, IC13, IC15, SVTP, CUTE数据集上进行评估，但是不管怎么训练svtr_tiny算法的测试评估精度都是60多一点，恳请各位大佬帮忙解答一下，可能是哪些原因导致的呢？

**Comment by wuxiaolianggit** [comment]
@tink2123 @sdcb @ZeyuChen @haobibo 各位大佬，可以帮我解答一下上面的问题吗？

**Comment by Haeppypuppy** [comment]
同問~
